mappings/lowfat-macula-hebrew.xquery

```diff
@@ -147,7 +147,16 @@ declare function local:attributes($node)
     $node/@state ! attribute state {lower-case(.)},
     $node/@stem ! attribute stem {lower-case(.)},
     $node/@person ! attribute person {lower-case(.)},
-    $node/@after
+    $node/@after,
+    $node/@Frame ! attribute frame {.},
+    $node/@Ref ! attribute ref {.},
```

Comment:
Since lowfat is using `@ref` for the verse reference, I would suggest making this attribute `@referent` so it's not confused with `@ref[erence]`.

Reply:
Is this the attribute that used to be called `@SubjRef``?  I'd like a name that is more specific about the meaning than just `@Ref`` for this variable, it's not a general reference for nodes in the tree.